addons/spreadsheet/static/src/global_filters/plugins/global_filters_ui_plugin.js

```diff
@@ -262,6 +265,46 @@ export class GlobalFiltersUIPlugin extends spreadsheet.UIPlugin {
         }
     }
 
+    /**
+     * Returns the possible values a text global filter can take
+     * if the values are restricted by a range of allowed values
+     * @param {string} filterId
+     * @returns {{value: any, formattedValue: string}[]}
```

Comment:
might as well return the values directly as strings, the only time we use them (in the side panel) we need string values, and we cast them as string